packages/nocodb/package.json

```diff
@@ -121,11 +121,13 @@
     "passport-jwt": "^4.0.0",
     "passport-local": "^1.0.0",
     "pg": "^8.3.0",
+    "request": "^2.88.2",
     "request-ip": "^2.1.3",
     "rmdir": "^1.2.0",
     "slash": "^3.0.0",
     "socket.io": "^4.4.1",
-    "sqlite3": "5.0.0",
+    "sqlite3": "5.1.2",
```

Comment:
Issue related to sqlite upgrade https://github.com/nocodb/nocodb/issues/1970 , will test the docker build later in linux and confirm thee same :+1:

Reply:
#### Verified docker in:

- [ ] Ubuntu
- [ ] MacOS
- [ ] Windows 

#### Verified executable in:

- [ ] Ubuntu
- [ ] MacOS
- [ ] Windows

Reply:
Executable failed in windows with the following error 

![image](https://user-images.githubusercontent.com/61551451/199915252-99601c0a-8184-4c16-a492-fc4581332e10.png)

Reply:
Probably we need to add binaries for the platform here - https://github.com/nocodb/nocodb-timely/tree/main/binaries/binding

Reply:
Executable failing in mac as well

<img width="1136" alt="Screenshot 2022-11-04 at 12 58 30 PM" src="https://user-images.githubusercontent.com/61551451/199917051-600f31aa-afcc-4c78-94e6-c0710660ad8d.png">

Reply:
Should be resolved with merge of @pranavxc 's PR

Reply:
yes 👍🏽